### ISSUE TITLE
Game info not displaying in Canterlot Siege

### ISSUE THREAD

**Issue by Computerdude77** [ruffle-rs/ruffle#8409]
### Describe the bug

In the game [Canterlot Siege](https://archive.org/details/canterlot_siege_by_futzi01-d5bfpyv), there is an issue with the in-game info (lives, waves, etc.). The numbers are not being shown at all, with the exception of the timer, which works perfectly. I am aware that there is also the same issue that all tower defense games seem to have at the moment that is likely related to #3839, but this is not specifically about that, and I thought I would mention it because it will likely be pointed out as well. Here is a screenshot demonstrating the issue:

![chrome_4r22g32tsD](https://user-images.githubusercontent.com/55714746/198820422-6ba90323-de50-4f2e-8c4c-6d759b12a79f.png)

### Expected behavior

This is a screenshot from native Flash, and you can see what is supposed to be displayed in the corner of the screen:

![flashplayer_32_sa_tPESrawyrN](https://user-images.githubusercontent.com/55714746/198820459-91645fac-e32d-443a-9177-4de78852ed63.png)

### Affected platform

Online demo

### Operating system

Windows 10 21H2

### Browser

Google Chrome 107

### Additional information

The version on Archive.org (2022-09-19) is older than the web demo this was tested on at the time of writing this (2022-10-28), but the issue is the same regardless of version.

**Comment by Lord-McSweeney** [comment]
Console repeatedly errors "Attempted to add frame scripts to non-MovieClip this!".

**Comment by Computerdude77** [comment]
As of the 2023-04-10 nightly, the issue with the UI numbers has been fixed. However, the issues documented above that involve the targeting still occur, so even though the original issue this was made for was fixed, I will leave this issue open until the above issues with the targeting are fixed too.

![chrome_e9RgCzn2ZF](https://user-images.githubusercontent.com/55714746/230843727-4b7488d8-95d9-4f6c-b5f1-a676a3a3785a.png)

**Comment by Computerdude77** [comment]
As of the 2023-05-29 nightly, the targeting bug has been fixed (I can even see the commit where it was fixed referenced above). The only other issue that seems to occur with this game is the music peaking sometimes when looping. I have documented a seemly identical issue with Spike Toss, so I am going to go ahead and close the issue, even with this sound bug. If it turns out to be an unrelated issue, someone can open up a separate issue for it.
